Explorer/Assets/DCL/Multiplayer/SDK/Systems/GlobalWorld/PlayerTransformPropagationSystem.cs

```diff
@@ -0,0 +1,52 @@
+using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using CrdtEcsBridge.Components;
+using CrdtEcsBridge.Components.Transform;
+using DCL.AvatarRendering.AvatarShape.UnityInterface;
+using DCL.Diagnostics;
+using DCL.Multiplayer.SDK.Components;
+using ECS.Abstract;
+using ECS.Groups;
+using ECS.LifeCycle.Components;
+
+namespace DCL.Multiplayer.SDK.Systems.GlobalWorld
+{
+    [UpdateInGroup(typeof(SyncedPreRenderingSystemGroup))]
+    [LogCategory(ReportCategory.MULTIPLAYER_SDK_PLAYER_TRANSFORM_DATA)]
+    public partial class PlayerTransformPropagationSystem : BaseUnityLoopSystem
+    {
+        public PlayerTransformPropagationSystem(World world) : base(world) { }
+
+        protected override void Update(float t)
+        {
+            PropagateTransformToSceneQuery(World!);
+        }
+
+        [Query]
+        [None(typeof(DeleteEntityIntention))]
+        private void PropagateTransformToScene(ref IAvatarView avatarView, ref PlayerCRDTEntity playerCRDTEntity)
```

Comment:
Can't we use `TransformComponent` instead of `IAvatarView`?

Reply:
not `TransformComponent` but yes `CharacterTransform`, I updated it at https://github.com/decentraland/unity-explorer/pull/1030/commits/b5c6df3d83c363bd98c1a924b75d0c43faf05742